cluster/plan.go

```diff
@@ -585,6 +587,14 @@ func (c *Cluster) BuildKubeProxyProcess(host *hosts.Host, prefixPath string, svc
 			CommandArgs[k] = v
 		}
 	}
+	// If cloudprovider is set to aws, set the bind address because the node will not be able to retrieve it's IP address because the nodename is set by AWS to internal DNS name
+	if c.CloudProvider.Name == AWSCloudProviderName {
```

Comment:
@superseb do we respect bind-addressed passed by the uservia cluster.yml modification here? It should take precedence over internally set bind-address

Reply:
The extraArgs functionality is still done after this section (https://github.com/rancher/rke/pull/1653/files#diff-2e9ef6d97acca90e8ed40f581b9f7264R633), so it should get overwritten if configured.